dashboard/app/views/levels/_summary.html.haml

```diff
@@ -39,7 +40,26 @@
   }
 
 
-#check-for-understanding
+#summary-content
+  #summary-top-links
+
+  .container#level-question
```

Comment:
As a follow up, could we move this to a place where it's shared with the level itself? Or is it too different for that to make sense?

Reply:
i think it's too different, but i'll look into it and see if i can make it work. it would be nice to not have the duplication

Reply:
No worries, just curious

Reply:
thanks! definitely a follow up task if it's possible at all!